.github/dependabot.yml

```diff
@@ -1,13 +1,12 @@
 version: 2
-
```

Comment:
Not a 100% sure about this format tbh, feels like blank lines should be kept?

Reply:
Haven’t looked into https://pkg.go.dev/gopkg.in/yaml.v3#Style might be something there

Reply:
Seems like there isnt an option for this

Reply:
See my latest commit for a potential solution

Reply:
:star_struck: